code/modules/mob/living/basic/space_fauna/regal_rat/regal_rat_ai.dm

```diff
@@ -1,7 +1,7 @@
 /datum/ai_controller/basic_controller/regal_rat
 	blackboard = list(
 		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic,
-		BB_BASIC_MOB_FLEEING = FALSE,
+		BB_BASIC_MOB_FLEEING = TRUE,
```

Comment:
oh that's my bad I thought something in the retaliate toflee stuff would modify this blackboard key